### PR TITLE
Modularize Expanded Options for Filter

### PR DESCRIPTION
**Pull request by samuy**
Modularized:
- Filter Options
- Filter Input Field
- Filter Stream Options
- Filter Time Input

### CODE REVIEW

**Review by shaisle — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -50,4 +50,8 @@ npm-debug.log.*
 *.scss.d.ts
 
 # SQLite3
-*.sqlite3
\ No newline at end of file
+*.sqlite3
+
+# Vim
+*.swp
+*.swo
```
Comment (noodlemonkey) [inline .gitignore:57]: why is this here

File: app/components/Filter.tsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+
+const Filter = ({ filterOptions, setFilterOptions }) => {
+  // A piece of local state to render the correct options to filter by
   const [selection, setSelection] = useState('');
 
-  //handles userInput into conditionally rendered input elements
-  //userinput changes with each key inputted
-  //on submit, sets filter options at the current text input
+  // Handles userInput into conditionally rendered input elements
+  // Userinput changes with each key inputted
```
Comment (noodlemonkey) [inline app/components/Filter.tsx:13]: what is this

**Review by noodlemonkey — APPROVED** [review]
